setup.cfg

```diff
@@ -90,9 +90,9 @@ parentdir_prefix =
 [flake8]
 max-line-length = 99
 doctests = True
-exclude=*build/
+exclude=*build/,*/_afq/*
```

Comment:
Is there still anything called `*afq*` anywhere in here?

Reply:
Yep, I placed the old code under ``dmriprep/workflows/_afq/`` for when we reinclude stuff as an _AFQ Plugin_. Let me know if that's fine with you all.